target/linux/realtek/dts-5.10/rtl838x.dtsi

```diff
@@ -262,10 +254,16 @@
 
 	sram0: sram@9f000000 {
 		compatible = "mmio-sram";
-		reg = <0x9f000000 0x10000>;
+		reg = <0x9f000000 0x20000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
-		ranges = <0 0x9f000000 0x10000>;
+		ranges = <0 0x9f000000 0x20000>;
+
+		sram_scratch: scratch-sram@0 {
+			compatible = "realtek,scratch";
+			reg = <0x0 0x20>;
+			pool;
+		};
```

Comment:
Any reason this has/needs a specific compatible?

Reply:
OK, this is something that would go into Documentation/devicetree/bindings/sram/sram.yaml then. Maybe add a label too.; something that indicates this is used by the clock driver to update the PLL registers: e.g.`label = "clock-config-update";` 

Can you use `protect-exec;` or is that not possible when also writing the blob from the driver?

Reply:
Unless I'm missing something, these new sram-scratch nodes also aren't used anywhere. Why are you adding them? If you need them for an upcoming feature, these should be included with those patches.

Reply:
They are used in the 930x assembler code (SRAM_BASE_UNCACHED) to "reset" SRAM communication. References in other DTs have been removed.